channels/az.m3u

```diff
@@ -1,25 +1,159 @@
+#EXTINF:-1 tvg-id="qafqaztv.az" tvg-name="QAFQAZTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/kJMCVkm.png" group-title="",QAFQAZ TV (720p)
+https://europe2.livetv.az/azerbaijan/qafqaztv/chunks.m3u8
+#EXTINF:-1 tvg-id="qafqaztv.az" tvg-name="QAFQAZTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/kJMCVkm.png" group-title="",QAFQAZ TV (720p)
+https://streams.livetv.az/azerbaijan2/qafqaztv/chunks.m3u8
+#EXTINF:-1 tvg-id="qebele.tv" tvg-name="QebeleTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/7ZXw0Zq.png" group-title="",Qebele TV (480p)
+https://qebele.tv/live/stream/index.m3u8
+#EXTINF:-1 tvg-id="realtv.az" tvg-name="REALTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/JOphTYk.png" group-title="",REAL TV (576p)
+http://185.129.1.242:80/realtv/mpegts
+#EXTINF:-1 tvg-id="realtv.az" tvg-name="REALTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/JOphTYk.png" group-title="",REAL TV (576p)
+https://europe2.livetv.az/azerbaijan/real/playlist.m3u8
+#EXTINF:-1 tvg-id="realtv.az" tvg-name="REALTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/JOphTYk.png" group-title="",REAL TV (576p)
+https://europe2.livetv.az/azerbaijan/real/chunks.m3u8
+#EXTINF:-1 tvg-id="spacetv.az" tvg-name="SPACETV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/gptHQJB.png" group-title="",SPACE TV (576p)
+https://europe2.livetv.az/azerbaijan/space_stream/playlist.m3u8
+#EXTINF:-1 tvg-id="spacetv.az" tvg-name="SPACETV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/gptHQJB.png" group-title="",SPACE TV (576p)
+https://europe2.livetv.az/azerbaijan/space_stream/chunks.m3u8
+#EXTINF:-1 tvg-id="kanalturan.com" tvg-name="TuranTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/bMywLye.png" group-title="",Turan TV (576p)
+https://donvolga.com/Turan-TV/video.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="ÜmüdTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/LTOGMmh.jpg" group-title="",Ümüd TV (720p)
+https://cdn-umudtv.yayin.com.tr/umudtv/ngrp:umudtv/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="ÜmüdTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/LTOGMmh.jpg" group-title="",Ümüd TV (720p)
+https://cdn1-umudtv.yayin.com.tr/umudtv/ngrp:umudtv/media_w1965476001_b797095_4372.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="ÜmüdTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/LTOGMmh.jpg" group-title="",Ümüd TV (360p)
+https://cdn-umudtv.yayin.com.tr/umudtv/ngrp:umudtv/chunklist_w1342030755_b1793849.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Xezer" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/6T4Q69m.jpg" group-title="",Xezer Tv (1080p)
+https://59c7ea6bec1c6.streamlock.net/live/xezertv.stream/playlist.m3u8?hash=d0f574372c36bc44c833002f87764f1d
```

Comment:
Link doesn't work.

Reply:
https://i.imgur.com/RQieGZe.png

Reply:
All I see is an error 403:
<img width="633" alt="Captura de pantalla 2021-04-13 a las 19 14 47" src="https://user-images.githubusercontent.com/7253922/114585637-85dade00-9c8c-11eb-8618-9918554cb992.png">
I guess the broadcast is just geo-blocked. So it will be enough if you add the label `[Geo-blocked]` to the channel name.